pkg/cloud/powervs.go

```diff
@@ -409,6 +409,8 @@ func getRegion(zone string) (region string, err error) {
 		region = "syd"
 	case strings.HasPrefix(zone, "mon"):
 		region = "mon"
+	case strings.HasPrefix(zone, "osa"):
```

Comment:
@Prajyot-Parab see is we need to update this code any more regions

Reply:
missed regions added now.